Root/OptimizationDump.cxx

```diff
@@ -35,7 +35,11 @@
 namespace HF = HelperFunctions;
 namespace VD = VariableDefinitions;
 
-#define ARRAY_INIT {-99, -99, -99, -99}
+
+#define N_LARGE_R 8
```

Comment:
Not going to do this -- I'll handle it with my `rc_enable` switch. We also don't need details for 8 large-R jets.. do we?